### PR TITLE
added new configs. Updated d-o-r

### PR DESCRIPTION
**Pull request by kprokofi**

### CODE REVIEW

**Comment by kprokofi** [comment]
run tests

**Comment by kprokofi** [comment]
run tests

**Comment by kprokofi** [comment]
run tests

**Comment by kprokofi** [comment]
run tests

**Comment by kprokofi** [comment]
run tests

**Comment by kprokofi** [comment]
run tests

File: models/image_classification/model_templates/custom-classification/README.md [hunk 1]
```diff
@@ -47,25 +47,27 @@ MobilenetV3:
...
 * [Sharpness aware minimization optimizer](https://www.semanticscholar.org/paper/Sharpness-Aware-Minimization-for-Efficiently-For%C3%AAt-Kleiner/bc52ab18399aaaf6b88c22ebc6e4a3caa99a2323)
+* [No bias decay method](https://www.semanticscholar.org/paper/Bag-of-Tricks-for-Image-Classification-with-Neural-He-Zhang/29309743870c825f9645a4803af727402462e513)
 
 All of the models were initially trained on [ImageNet](http://image-net.org/challenges/LSVRC/2012/), but can be trained from scratch or fine tuned to classify arbitrary images.
 
 Information about LR Finder:
 
-There are two options for learning rate finder avalaible: smart brute force (more accurate, but long) and by fast.ai approach imported from [torch-lr-finder](https://github.com/davidtvs/pytorch-lr-finder/blob/master/torch_lr_finder/lr_finder.py) with some modifications.
+There are three options for learning rate finder avalaible: [Optuna's](https://arxiv.org/abs/1907.10902) grid search, TPE and by fast.ai approach imported from [torch-lr-finder](https://github.com/davidtvs/pytorch-lr-finder/blob/master/torch_lr_finder/lr_finder.py) with some modifications.
```
Comment (sovrasov) [inline models/image_classification/model_templates/custom-classification/README.md:64]: There are three algorithms for estimating an optimal learning rate available

**Comment by kprokofi** [comment]
@Ilya-Krylov , the tests on the image classification are OK and for other domains except for the Action Recognition. But the @evgeny-izutov is on the vacation now, What should we do?

**Review by sovrasov — APPROVED** [review]

**Comment by sovrasov** [comment]
The failure is related to mmcv installation. I also encountered this problem, but after several attempts found the fail-safe version (mmcv==1.3.8, whereas the latest problematic version is 1.3.10).

**Comment by kprokofi** [comment]
run tests

File: ote/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 numpy==1.19.5
-mmcv==1.3.9
```
Comment (sovrasov) [inline ote/requirements.txt:2]: 1.3.9 also works, seems like only the latest .10 is broken

File: models/action_recognition/requirements.txt [hunk 3]
```diff
@@ -1,4 +1,5 @@
 -r ../../external/mmaction2/requirements.txt
 joblib
 opencv-python==4.1.0.25
+mmcv==1.3.8
```
Comment (Ilya-Krylov) [inline models/action_recognition/requirements.txt:4]: remove this change please

File: ote/requirements.txt (hunk 2 above)
Comment (Ilya-Krylov) [reply ote/requirements.txt:2]: revert this change please

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

**Comment by Ilya-Krylov** [comment]
Could you please update mmaction2 submodule to point to the latest state of 'ote' branch there. I've updated requirements https://github.com/openvinotoolkit/mmaction2/pull/4

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Review by Ilya-Krylov — APPROVED** [review]

**Pull request merged**